client/src/components/FormBasics.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import FormProfiles from "./FormProfiles";
+
+export default function FormBasics() {
+  return (
+    <section>
+      <h3>Basics:</h3>
+      <div style={{ display: "flex" }}>
```

Comment:
Styling in CSS is more performative than JS, especially when it could easily be selected.